src/bindings/pygaia/scripts/classification/classification_project_template.yaml

```diff
@@ -38,12 +51,15 @@ preprocessing:
                                                     '*.min', '*.max', '*cov',
                                                     'tonal.thpcp', # because of division by zero
                                                     'lowlevel.spectral_energyband_high.*', # 0 for low samplerate
-                                                    'lowlevel.silence_rate*' # funky behavior in general
+                                                    'lowlevel.silence_rate*', # funky behavior in general
```

Comment:
The remove algorim fails if it can't find these features to remove. This seems weird, as the end goal is to remove it anyway.
The algorithm has an option failOnUnmatched, which defaults to true. Perhaps we can set this to false, or add the argument to the function definition in the project file